src/apps/testapps/testPolygon.c

```diff
@@ -169,4 +204,35 @@ TEST(bboxesFromGeoPolygonHole) {
     free(result);
 }
 
+TEST(bboxFromLinkedGeoLoop) {
+    const GeoCoord verts[] = {{0.8, 0.3}, {0.7, 0.6}, {1.1, 0.7}, {1.0, 0.2}};
+
+    LinkedGeoLoop loop;
+    initLinkedLoop(&loop);
+
+    for (int i = 0; i < 4; i++) {
+        addLinkedCoord(&loop, &verts[i]);
+    }
+
+    const BBox expected = {1.1, 0.7, 0.7, 0.2};
+
+    BBox result;
+    bboxFromLinkedGeoLoop(&loop, &result);
+    t_assert(bboxEquals(&result, &expected), "Got expected bbox");
+
+    destroyLinkedGeoLoop(&loop);
+}
+
+TEST(bboxFromLinkedGeoLoopNoVertices) {
+    LinkedGeoLoop loop;
+    initLinkedLoop(&loop);
```

Comment:
Does this need a `destroyLinkedGeoLoop`?

Reply:
`destroyLinkedGeoLoop` is used to free the memory for the `LinkedGeoCoord` nodes in the loop, but this loop doesn't have any.

Reply:
I'd suggest adding it even though it is a no-op, since it matches the contract of LinkedGeoLoop.

Reply:
Agreed, done.